Codeinwp/neve
# Add BF deal

### Summary


Add Black Friday event.

### Will affect visual aspect of the product

YES

### Screenshots 

![image](https://github.com/Codeinwp/neve/assets/17597852/2d744596-e41e-4647-8752-396768f5581c)

![image](https://github.com/Codeinwp/neve/assets/17597852/492d0f0d-fb2b-4754-b7bb-d991188094a8)

![image](https://github.com/Codeinwp/neve/assets/17597852/dff1ae2b-772c-422d-a051-190f8670e8ed)


### Test instructions


1. Check if the banners appear with the correct link/URL. More about this here: https://github.com/Codeinwp/themeisle/issues/1575#issuecomment-1757321452
2. By using all three plugins (Neve, Otter, Feedzy), check the admin notice (the small one) priority. There should be notice only for the highest priority product. The order is Neve > Otter > Feedzy. More on this here: https://github.com/Codeinwp/themeisle/issues/1575#issuecomment-1761388729

The big banners will have a dynamic text that shows the remaining time in the top-left corner. For the start is set to an early one to facilitate the test (it will be changed after the approval). ~The time interval set for testing is: 2023-10-10 00:00:00 and 2023-11-27 23:59:00~. If possible, try to change the server time to see if the banner is working correctly when the time is over.

The PRs are:
- Neve: https://github.com/Codeinwp/neve/pull/4121
- Otter: https://github.com/Codeinwp/otter-blocks/pull/1922
- Feedzy: https://github.com/Codeinwp/feedzy-rss-feeds/pull/805

## Check before Pull Request is ready:



Closes #.
